api/app/src/main/kotlin/packit/service/RunnerService.kt

```diff
@@ -66,4 +69,21 @@ class BaseRunnerService(
         runInfoRepository.save(runInfo)
         return res.taskId
     }
+
+    override fun getTaskStatus(taskId: String): RunInfoDto
+    {
+        val taskStatus = orderlyRunnerClient.getTaskStatuses(listOf(taskId), true)[0]
+
+        val runInfo = runInfoRepository.findByTaskId(taskId)!!
+        runInfo.timeQueued = taskStatus.timeQueued
```

Comment:
```suggestion
        runInfo.apply {
            timeQueued = taskStatus.timeQueued
            timeStarted = taskStatus.timeStarted
            timeCompleted = taskStatus.timeComplete
            logs = taskStatus.logs
            status = taskStatus.status
            packetId = taskStatus.packetId
            queuePosition = taskStatus.queuePosition
        }
```
a little bit nicer syntatically :smile:

Reply:
thats pretty neat